pyproject.toml

```diff
@@ -18,6 +18,7 @@
 
   [project.optional-dependencies]
     test = ["pytest", "numpy>=1.5.0", "pytest-subprocess==1.5.0"]
+    types = ["mypy", "types-PyYAML", "types-redis", "types-requests", "types-ujson"]
```

Comment:
types-redis and types-ujson? are we using them?

Reply:
Yes - indirectly and we need to pip install both of them them (we get errors if we don't). Both provide stubs for dependencies of requests-cache (which we use directly). My guess is that all four of these requirements stub packages will go away in time because they'll get merged with the main packages.